configs/prod/roles/drift.json

```diff
@@ -1,14 +1,20 @@
 {
   "roles": [
     {
-      "name": "Drift analysis administrator",
-      "description": "Perform any available operation against any Drift Analysis resource.",
+      "name": "Drift viewer",
```

Comment:
I think this is incorrect.

1. The idea is to retain `Drift analysis administrator` and make it `"admin_default": true,` rather than `"platform_default": true,`  see example: https://github.com/RedHatInsights/rbac-config/pull/264/files
2. Perhaps create a separate `Drift viewer` permission like shown in the above link and add all drift related read-only permissions to those.

cc @jeromemarc

Reply:
@sthirugn I made the changes!

Thanks